source/sync/data-model/update-schema.txt

```diff
@@ -270,9 +272,9 @@ How to Remove an Object Type
 
 A breaking server-side schema change requires you to :ref:`terminate sync 
 <terminating-realm-sync>` in the backend, and then :ref:`re-enable 
-<enable-sync>` sync. This triggers a :ref:`client reset <client-resets>`, 
-which can result in client-side data loss. As a result, we do not recommend
-making breaking server-side schema changes.
+<enable-sync>` sync. Breaking schema changes prevent applications from 
```

Comment:
This page is written so devs can select the type of change they want to make and see how to deal with it. I don't expect most people will read the page from top to bottom; I expect they'll see the table, go "Oh, I want to change this property from optional to required" (for example) and then jump to that section and read it. All of the information that they need should be contained in the section that's relevant to them, because we can't assume they'll read any other sections of the page.

Reply:
Good point